proto/zitadel/resources/action/v3alpha/target.proto

```diff
@@ -84,6 +89,8 @@ message PatchTarget {
       max_length: 1000
     }
   ];
+  // Regenerate the key used for signing and checking the payload sent to the target.
+  optional bool regenerate_signing_key = 7;
```

Comment:
as just discussed, a Duration would not brake in case we want to have graceful rotation